sycl/test-e2e/syclcompat/memory/memory_management_test3_usmnone.cpp

```diff
@@ -0,0 +1,601 @@
+// ====------ memory_management_test3_usmnone.cpp---------- -*- C++ -* ----===////
```

Comment:
This test is probably the one that should check `syclcompat::wait_and_free`

Reply:
Done!